downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc

```diff
@@ -1,34 +1,34 @@
-
+// Module included in the following assemblies:
+// assembly-disconnected-installation.adoc
 
 [id="con-aap-installation-on-disconnected-rhel_{context}"]
 
-
-= {PlatformNameShort} installation on disconnected RHEL
+= Installing {PlatformNameShort} on disconnected RHEL
 
 [role="_abstract"]
-Install Ansible Automation Platform (AAP) automation controller and a private automation hub, with an installer-managed database located on the automation controller without an Internet connection.
+You can install {PlatformNameShort} {ControllerName} and a {PrivateHubName} without an internet connection using the installer-managed database located on the {ControllerName}.
 
 == Prerequisites
 
-To install AAP on a disconnected network, complete the following prerequisites:
+To install {PlatformNameShort} on a disconnected network, complete the following prerequisites:
 
 . Create a subscription manifest.
-. Download the AAP setup bundle.
-. Create DNS records for the automation controller and private automation hub servers.
+. Download the {PlatformNameShort} setup bundle.
+. Create DNS records for the {ControllerName} and {PrivateHubName} servers.
 
-NOTE: The setup bundle includes additional components that make installing AAP easier in a disconnected environment. These include the AAP RPMs and the default execution environment (EE) images.
+NOTE: The setup bundle includes additional components that make installing {PlatformNameShort} easier in a disconnected environment. These include the {PlatformNameShort} Red Hat package managers (RPMs) and the default execution environment (EE) images.
```

Comment:
This is secondary to all other primary updates, but this note should follow the syntax for Admonitions: 

```suggestion
[NOTE]
====
The setup bundle includes additional components that make installing {PlatformNameShort} easier in a disconnected environment. These include the {PlatformNameShort} Red Hat package managers (RPMs) and the default execution environment (EE) images.
====
```

Reply:
(I removed the note formatting and just made this section part of the concept module abstract)